packages/cli/generators/extension/index.js

```diff
@@ -69,6 +69,14 @@ module.exports = class ExtensionGenerator extends ProjectGenerator {
   }
 
   scaffold() {
+    if (this.projectInfo) {
+      const titleCase =
+        this.projectInfo.name.charAt(0).toUpperCase() +
+        this.projectInfo.name.slice(1);
+      this.projectInfo.optionsInterface = `${titleCase}Options`;
+      this.projectInfo.bindingsNamespace = `${titleCase}Bindings`;
+      this.projectInfo.defaultOptions = `DEFAULT_${this.projectInfo.name.toUpperCase()}_OPTIONS`;
```

Comment:
Please make sure we are handling multi-word names correctly and consistently with other places that are building PascalCase and camelCase identifiers.

A test case to consider: `projectInfo.name` is `todo-demo`. I expect the following strings to be used:
- options interface: `TodoDemoOptions`
- bindings namespace: `TodoDemoBindings`
- default options: `DEFAULT_TODO_DEMO_OPTIONS`

Please check how is `lb4 app` generator dealing with multi-word app names and apply the same solution for extensions. If the app generators behaves differently from what I described above, then please follow what the app generator is doing.

Reply:
`lb4 app` is not doing the upperase-underscore thing (eg: `DEFAULT_TODO_DEMO_OPTIONS`), generating it this way for extension:

```ts
      const uppercaseUnderscore = this.projectInfo.name
        .toUpperCase()
        .replace(/\W/g, '_');
      this.projectInfo.defaultOptions = `DEFAULT_${uppercaseUnderscore}_OPTIONS`;
```

Rest, updated to be like `lb4 app`.